dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,339 @@
+                  </FormKit>
+
+                  <FormKit name="eligibilityKpis" type="group">
+                    <FormKit :name="euTaxonomyKPIsModel?.companyTypeToEligibilityKpis[copanyType.value]" type="group">
+                      <div
+                        v-for="kpiTypeEligibility of euTaxonomyKPIsModel.eligibilityKpis"
+                        :key="kpiTypeEligibility"
+                        :data-test="kpiTypeEligibility"
+                        class="uploadFormSection"
+                      >
+                        <div class="col-9 formFields">
+                          <FormKit :name="kpiTypeEligibility" type="group">
+                            <div class="form-field">
+                              <h3>
+                                {{ euTaxonomyKpiNameMappings[kpiTypeEligibility] ?? "" }}
+                              </h3>
+                              <KPIfieldSet
+                                :name="kpiTypeEligibility ?? ''"
+                                :kpiInfoMappings="euTaxonomyKpiInfoMappings"
+                                :kpiNameMappings="euTaxonomyKpiNameMappings"
+                                :reportsName="namesOfFilesToUpload"
+                              />
+                            </div>
+                          </FormKit>
+                        </div>
+                      </div>
+                    </FormKit>
+                  </FormKit>
+                </div>
+              </FormKit>
+
+              <!--------- SUBMIT --------->
+
+              <div class="uploadFormSection grid">
+                <div class="col-3"></div>
+
+                <div class="col-9">
+                  <PrimeButton data-test="submitButton" type="submit" label="SUBMIT FORM" />
+                </div>
               </div>
-            </template>
-          </FormKit>
-          <FormKit type="group" name="creditInstitutionKpis" label="Credit Institution KPIs">
-            <h4>Credit Institution KPIs</h4>
-            <div name="creditInstitutionKpis">
-              <DataPointFormElement name="tradingPortfolio" label="Trading Portfolio" />
-              <DataPointFormElement name="interbankLoans" label="Interbank Loans" />
-              <DataPointFormElement
-                name="tradingPortfolioAndInterbankLoans"
-                label="Trading Portfolio and Interbank Loans (combined)"
-              />
-              <DataPointFormElement name="greenAssetRatio" label="Green asset ratio" />
             </div>
           </FormKit>
-          <FormKit type="group" name="insuranceKpis" label="Insurance KPIs">
-            <h4>Insurance KPIs</h4>
-            <DataPointFormElement
-              name="taxonomyEligibleNonLifeInsuranceActivities"
-              label="Taxonomy Eligible non Life Insurance Activities"
+          <template v-if="postEuTaxonomyDataForFinancialsProcessed">
+            <SuccessUpload
+              v-if="postEuTaxonomyDataForFinancialsResponse"
```

Comment:
A better check might be just looking for `postEuTaxonomyDataForFinancialsResponse?.status === 200`.
Then you can also remove the `this.postEuTaxonomyDataForFinancialsResponse = null` from the `catch` block in the method `postEuTaxonomyDataForFinancials()`

Reply:
ok